app/main.py

```diff
@@ -1,3 +1,70 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> float:
```

Comment:
The `__init__` method should not have a return type annotation of `float`. It should be `None` since constructors do not return a value.